ts/njs_core.d.ts

```diff
@@ -606,3 +606,45 @@ interface NjsProcess {
 }
 
 declare const process: NjsProcess;
+
+/**
+ * A value returned by `setTimeout()` and `setImmediate()` functions. It's an positive integer now,
+ * but this may be changed in future, so it should be treated as an opaque value.
+ */
+type TimerHandle = number & { readonly '': unique symbol };
+
+/**
+ * Schedules the "immediate" execution of the given function after I/O events' callbacks.
+ *
```

Comment:
Actually, after promise microtasks
```js
>> setImmediate(() => console.log('two')); require('fs').access('/', () => console.log('three')); Promise.resolve().then(() => console.log('one'));
Promise {
 
}
one
two
three
```

Reply:
I’ve copied this description from Node’s documentation. Microtask queue is also Node’s term. Does njs use the same terminology for this? @xeioex

Reply:
@jirutka  I would leave the description as is (we do not want to be too specific here, because it is an implementation detail).

>Does njs use the same terminology for this?

We generally prefer to use the same terminology to avoid confusion, but again, here, IMHO, it is too low level.